utils/utils.js

```diff
@@ -134,8 +134,8 @@ function handleError(error, analyticsInstance) {
       // Discard all the non-script loading errors
       if (error.target && error.target.localName !== "script") return;
 
-      // Discard errors of scripts that are not loaded by the SDK
-      if (error.target.dataset && error.target.dataset.loader !== LOAD_ORIGIN)
+      // Discard all script errors unless they are of integration SDK scripts
+      if (error.target.dataset && (error.target.dataset.loader !== LOAD_ORIGIN || error.target.dataset.isNonNativeSDK === defaultNonNativeSDKState))
```

Comment:
```suggestion
      if (error.target.dataset && (error.target.dataset.loader !== LOAD_ORIGIN || error.target.dataset.isNonNativeSDK !== true))
```

Reply:
element.dataset.property converts the value to string.

Reply:
Addressed